docs/disk-expansion.md

```diff
@@ -17,3 +19,34 @@ spec:
 Enabling this feature does two things:
 - Notify the virtual machine about size changes
 - If the disk is a Filesystem PVC, the matching file is expanded to the remaining size (while reserving some space for file system overhead).
+
+## Usage
+
+To expand a disk, edit the matching PersistentVolumeClaim:
+
+`kubectl edit pvc my-disk-pvc`
+
+And increase the spec.resource.requests.storage to a larger size.
+A running VMI will be notified that the disk has been expanded.
```

Comment:
Would it make sense to explain how is the running VMI notified ? I think you should at least mention over which medium the notification is sent, along with what is the purpose of the notification.

Reply:
so I've had a look at the exact mechanism, and for SCSI it sends a "CAPACITY DATA HAS CHANGED" and for VirtIO, updates the config generation and sends a device configuration interrupt. I feel like writing this may be a bit too low level to mention in this context.
The libvirt docs [appear to be lacking](https://libvirt.org/html/libvirt-libvirt-domain.html#virDomainBlockResize), too.

Reply:
I'm just pointing out I know a notification exists, but don't know what listens to it, nor over what, nor what it is used for. 

Might as well either add info, or remove the mention of this altogether.

Reply:
I've written slightly more